README.md

```diff
@@ -28,26 +28,15 @@ var es = new saltpack.stream.EncryptStream({
 var ds = new saltpack.stream.DecryptStream({decryptor: bob, do_armoring: true})
 
 //register error listeners
-es.on('error', (err) -> throw err)
-ds.on('error', (err) -> throw err)
-
-//encrypt from stdin, write to a file
-var fs = require("fs");
-var file = fs.createWriteStream("/path/to/file");
-
-//pipe the EncryptStream into the file
-process.stdin.pipe(es);
-//currently, it's not possible to chain pipe calls with {Encrypt,Decrypt}Streams, i.e.
-//process.stdin.pipe(es).pipe(file) - this change may come down the line,
-//but for now we just have to use the extra line.
-es.pipe(file);
-file.on('close', () -> es.end())
-
-//decrypt from a file, write to stdout
-file = fs.createReadStream("/path/to/file");
-file.pipe(ds)
-ds.pipe(process.stdin)
-file.on('close', () -> ds.end())
+es.on('error', function (err) { throw err; })
+ds.on('error', function (err) { throw err; })
+
+//roundtrip encrypt/decrypt from stdin to stdout
+process.stdin.pipe(es)
+//{Encrypt,Decrypt}Streams don't yet know how to accept pipes, so we have to pipe to
+//their .first_stream.
+es.pipe(ds.first_stream);
+ds.pipe(process.stdout)
```

Comment:
Wait, do you need to refresh your diff? Line 34 is `//roundtrip encrypt/decrypt from stdin to stdout`